.claude/agents/docs-agent.md

```
@@ -0,0 +1,198 @@
+
+# Verify make targets exist
+grep '```bash' *.md | grep 'make ' | sed 's/.*make \([a-z-]*\).*/\1/' | sort -u
+
+# Check for dead links (manual review)
+grep -r '\[.*\](' *.md docs/*.md
+```
+
+## Output Format
+
+When updating docs, report:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)